src/dash/Profile.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { setUserAction } from '../redux/userActions.js';
+import axios from 'axios';
+
+class Profile extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            first_name: 'Sammy',
+            last_name: 'Baek',
+            email: 'asdf@email.com',
+            join_date: new Date(),
+            bio: 'Currently this is hardcoded json. The username and password are from the user. Still need to make responsive',
+            classroom: 'asdf1234',
+            grade_level: 9,
+            is_edit: false,
+            button_text: 'Edit',
+        }
+        this.handleEdit = this.handleEdit.bind(this);
+        this.handleBioChange = this.handleBioChange.bind(this);
+        this.handleEmailChange = this.handleEmailChange.bind(this);
+    }
+
+    componentDidMount() {
+        var self = this;
+        axios.get('/user')
+            .then(function (response) {
+                if (response.data !== '') {
```

Comment:
If you're just checking to see if this is not an empty string you may be able to do something like `if (response.data)` as opposed to `if (respons.data !== '')` because of the fact that an empty string is falsey.